app/actions/views/channel.js

```diff
@@ -149,13 +149,15 @@ export function loadPostsIfNecessaryWithRetry(channelId) {
 
         const time = Date.now();
 
-        let loadMorePostsVisible;
+        let loadMorePostsVisible = true;
```

Comment:
Are these changes related to this PR?

Reply:
nope but it was throwing errors if there was no connection to the server